src/features/import/l10n/messageIds.ts

```diff
@@ -63,6 +63,15 @@ export default makeMessages('feat.import', {
         ),
         value: m('Value'),
       },
+      genders: {
+        label: m('Gender'),
+        selectLabels: {
+          f: m('female'),
+          m: m('male'),
+          o: m('other'),
+          unknown: m('unspecified'),
```

Comment:
Tiny thing here - still in need of uppercase on the options, and I think "unspecified" should be changed to "unknown" as the value! :D

Reply:
Oops, it did feel like I was forgetting something 🙈